forem/forem
# ✂️✂️✂️  Part 1: Remove Sponsorship

## What type of PR is this? (check all applicable)


## Description
In this PR, I remove the sponsorship code from views, i8n, controllers, unused decorators, unused slack messengers and more.   

In a subsequent PR I'll remove the code from models, the database and do some more cleanup to the css and i8n files. The reason i separated it out like this is:
- I did not want this PR to be **even** bigger than it is
-  if I've missed anything in the views, it is less likely to break because the model and data are still in tact. 

## Related Tickets & Documents



- Related Issue #https://github.com/forem/forem/issues/18486
- Closes #

## QA Instructions, Screenshots, Recordings

This will be a bit difficult to test, but make sure that you have some sponsorships set up and live and then play around on the interface to make sure nothing breaks.

### UI accessibility concerns?

None

## Added/updated tests?

- [x] No, and this is why: I've removed code

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/bq6F8QYqBU7Yc/giphy.gif)
